src/backend/parser/parse_clause.c

```diff
@@ -53,6 +53,7 @@
 #include "utils/syscache.h"
 
 tle_name_comparison_hook_type  tle_name_comparison_hook = NULL;
+NamespaceStack *set_op_ns_stack = NULL;
```

Comment:
This value can be moved into the babel 
PG runtime can use hook to read/write to set_op_ns_stack